server/controllers/UserController.js

```diff
@@ -116,7 +116,7 @@ class UserController {
           token: userToken,
         });
       })
-      .catch(err => (err.message));
+      .catch((err) => { console.log(err.message); });
```

Comment:
You forgot to remove the console.log on line 199 of UserController.js. Please, check the file for any other occurrence. Thank you